interface/resources/qml/hifi/dialogs/security/Security.qml

```diff
@@ -333,6 +333,8 @@ Rectangle {
             anchors.left: parent.left;
             anchors.right: parent.right;
             height: childrenRect.height;
+            // FIXME: Reuse or remove wallet-related code.
+            visible: false
```

Comment:
Seems to be missing a `;`, looking at the rest of the code.

Reply:
QML is best written without ";"s ... alas some QMLers have used them.